### ISSUE TITLE
Snapshot filesize

### ISSUE THREAD

**Issue by hulkhaugen** [FaserF/hassio-addons#20]
This is less of an issue and more a question. I'm currently trying out this addon which works well, but I noticed that it significantly adds to the size of my automated snapshots, about 86 MB in my case to be precise. It seems that the actual add-on is included in the backup, and not just the config files. I don't know if this is an issue, or just the way it works.

I must admit that I know next-to-nothing about how these containers are structured. Personally, I only need to host a static website with html, css and a couple of js-files, so I'm thinking about copying your project and trim it down. Looking at your dockerfile, it seems that I should be able to remove pretty much all packages apart from apache2, and that I could also build from alpine. Additionally, remove all references to php  in run.sh and config.json. Would you mind that, and am I on to something?

**Comment by FaserF** [comment]
I have to admit I haven’t optimized the addon for backups. Personally I recommend excluding my addon from backups, and be sure to backup /share instead. As all the important stuff is in the share folder and not inside this addon

the problem is That I had to find a solution for people just wanting a simple HTML Page (where you are right, your dockerfile would then only need apache2 within alpine) and other people from older issues here on GitHub who wanted to be able to use more advanced php packages with MariaDB support for example. So you are True, the addon is definitive „blown up“.

so in conclusion the best for you would be to 
1. exclude apache2 from your backups and add /share
2. To copy my addon and slim it down to your needs

**Comment by FaserF** [comment]
This addon now has three variants. Two of them are "minimal" variants since V2.0.0
Maybe thats a way to go for you if you dont need the most PHP7 extensions.
